main.go

```diff
@@ -120,6 +121,8 @@ func main() {
 	flag.BoolVar(&opts.GetConnzDetailed, "connz_detailed", false,
 		"Get detailed connection metrics for each client. Enables flag `connz` implicitly.")
 	flag.BoolVar(&opts.GetHealthz, "healthz", false, "Get health metrics.")
+	flag.BoolVar(&opts.GetHealthzJsEnabledOnly, "healthz_js_enabled_only", false, "Get health metrics with js-enabled-only=true.")
+	flag.BoolVar(&opts.GetHealthzJsServerOnly, "healthz_js_server_only", false, "Get health metrics with js-server-only=true.")
```

Comment:
these look mutually exclusive. should we check for that?

Reply:
Well in fact they are not: you could want to get the 3 different metrics exposed to your monitoring depending on the use case. They are exposed on different metrics name so they can be enables simultaneously.
We are using both `healthz_js_server_only` to check that servers are globally available and `healthz` to monitor more general health (as this includes all jetstream stream availability it may go down when creating many streams while the servers are syncing and this led to false alerts on our monitoring during test runs)

Reply:
but you'd have to run multiple exporters for that, no?
I was actually referring to this bit:

https://github.com/nats-io/prometheus-nats-exporter/pull/309/files#diff-62fd247a63ada12ff62981409b10f487239ff90cdd055af67e3bf3c1da79bb09R54-R60

```go
	healthzURLPathAndQuerryArgs := healthzEndpoint
	switch endpoint {
	case healthzJsEnabledOnlyEndpoint:
		healthzURLPathAndQuerryArgs = healthzEndpoint + "?js-enabled-only=true"
	case healthzJsServerOnlyEndpoint:
		healthzURLPathAndQuerryArgs = healthzEndpoint + "?js-server-only=true"
	}
```

so it's not possible in the same process unless I'm missing something.

(btw not a biggy just wanted to clarify)

Reply:
It's not possible by the same "collector" but the options creates one for each variant there:

https://github.com/nats-io/prometheus-nats-exporter/pull/309/files#diff-c722ac09a12b0798d4b56729c7f9c7236651f195ea1d3ee985db75543d5ce219R198-R203

So a single `prometheus-nats-exporter` process can expose the 3 metrics but it will do 3 requests to the nats server, one for each enabled health* option